Azure-Samples/digital-twins-samples-csharp
# Add the new UDF and change computed property name

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] New sample or new feature within a sample
[ ] Code style update (formatting, naming)
[ ] Refactoring (no functional changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
manually
Also verify anything you think is relevant by manually running the sample(s)
```
cd <folder>
dotnet run
```

## Other Information
